lib/vmdb/loggers/api_logger.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "multicast_logger"
```

Comment:
I think it's not autoloading because you didn't fully qualify the logger.

```ruby
module Vmdb
  module Loggers
    class ApiLogger < MulticastLogger
```